### ISSUE TITLE
CSV Import - New Payees Not Created on import

### ISSUE THREAD

**Issue by gmulhall** [moneymanagerex/moneymanagerex#6881]
**Describe the bug**
I just switched from QIF import to CSV import after switching my data clean from MSExcel to Google Sheets.
The CSV import works fine (CSV file has comma's only - no quotes between fields) - except that the new Payees were not imported whereas with the QIF import you were aleted to new payees and categories and could continue with the import or not.
I don't know if new categories are imported but I can try this if needed.

**To Reproduce**
1. Go to File->Import CSV File ensuring there is a payee not yet in the MMEx SQlite Db
2. Locate the imported Tx
3. Observe that the Payee field is blank

**Expected behavior**
Expect the CSV import to act as the Qif file does ie offer to create new payees and new categories on import or cancel the import  

**Screenshots**
N/A

**Device Information:**
 Operating System
	Windows 10 Pro 64-bit
CPU
	AMD FX-6300	24 °C
	Vishera 32nm Technology
RAM
	8.00GB Single-Channel DDR3 @ 668MHz (11-11-11-28)
Motherboard
	ASUSTeK COMPUTER INC. M5A97 R2.0 (Socket 942)	30 °C
Graphics
	ASUS VP278 (1920x1080@59Hz)
	2047MB NVIDIA GeForce GT 710 (ASUStek Computer Inc)	42 °C
Storage
	111GB OCZ-ARC100 (SATA (SSD))
	1863GB Seagate ST2000DM001-9YN164 (SATA )	33 °C
	7GB SanDisk Cruzer Glide USB Device (USB )
Optical Drives
	HL-DT-ST DVDRAM GH24NSC0
Audio
	NVIDIA High Definition Audio

**Additional context**
I can provide a csv file if wanted

**Comment by vomikan** [comment]
I remind you that the latest release of rpilax 1.8.0.
Available for download here: https://github.com/moneymanagerex/moneymanagerex/releases/tag/v1.8.0

**Comment by gmulhall** [comment]
I am running on - 
Money Manager Ex
• Version: 1.8.0 64-bit
• Built on: Apr 29 2024 08:08:39
• Database version: 19 (aes128cbc)
• Git commit: 3325705 (2024-04-29)

**Comment by vomikan** [comment]
> I can provide a csv file if wanted

Could you please upload the file here? 
One line with header should be enough.

**Comment by gmulhall** [comment]
[MMEx - Export.csv](https://github.com/user-attachments/files/17020827/MMEx.-.Export.csv)

I trust this has loaded OK

Thanks again,

Geoff

**Comment by vomikan** [comment]
I've got the message:
Line 61: The opening date for the account is later than the date of this transaction

It seems to me that I have already solved such a problem

**Comment by vomikan** [comment]
The file imported correctly
![image](https://github.com/user-attachments/assets/1ac11a48-2f79-4722-818f-06aceabcc502)

**Comment by vomikan** [comment]
The import settins
![image](https://github.com/user-attachments/assets/57ee6a00-e921-4fb5-a185-4bc53a7d4e9f)

**Comment by gmulhall** [comment]
Hi,

Re:

**"I've got the message:
Line 61: The opening date for the account is later than the date of this transaction
It seems to me that I have already solved such a problem"**

I expect you have created a new account to test the import and that therefore the account opening date is after any of the transactions being loaded and they therefore reject with the message you received.

I think that to properly test the issue the account opening date needs to be set so say 1/1/2024 ie before any of the transactions in my csv file. Additionally some of the payees in the CSV should be manually loaded to the database. Then you'll be able to see which transactions appear with Payees as expected and the remaining transactions should appear but all Payee fields for those transactions will be blank I expect. This should replicate my experience.

Thanks again,

Geoff

**Comment by gmulhall** [comment]
Hi,
I now see you have imported the file. I'll so retest at my end. 
Can I ask if you receives the mesages warning about new categories &/or Payees as with a Qif file ?
Thanks again,
Geoff

**Comment by vomikan** [comment]
Unfortunatly I can't reproduce this issue again.

**Comment by gmulhall** [comment]
So I retested and everything came in OK - so I must have had some blank payees in my input file when I first did the import.
So apologies for wasting time.
Two things come out of this - 
1. That the messages about importing new categories or payees does not appear - so there's some inconsistency.
2. There is no warning that there are blank Payees in the import - which I think really ther should be.

Anyhow thanks for looking at this. It is appreciated.
